src/load_wav.cpp

```diff
@@ -121,7 +121,7 @@ BOOL CSoundFile::ReadWav(const BYTE *lpStream, DWORD dwMemLength)
 			signed char *psrc = (signed char *)(lpStream+dwMemPos+8+nChn*slsize+slsize-2);
 			for (UINT i=0; i<len; i++)
 			{
-				p[i] = *((signed short *)psrc);
+				p[i] = READ_LE16(psrc);
```

Comment:
I get the following compile error:
```
load_wav.cpp:124:50: error: invalid conversion from 'signed char*' to 'LPCBYTE' {aka 'const unsigned char*'} [-fpermissive]
  124 |                                 p[i] = READ_LE16(psrc);
      |                                                  ^~~~
      |                                                  |
      |                                                  signed char*
In file included from load_wav.cpp:8:
../src/libmodplug/sndfile.h:960:31: note:   initializing argument 1 of 'WORD READ_LE16(LPCBYTE)'
  960 | inline WORD READ_LE16(LPCBYTE b) {
      |                       ~~~~~~~~^
make[2]: *** [Makefile:573: load_wav.lo] Error 1
```

Reply:
> I get the following compile error:

My mistake. Fixed now.

> All other changes looked OK.

OK.

> Have the mmcmp.cpp portions been tested?

Tested by playing back a few mmcp'ed modules.